### PR TITLE
Document ReferenceRect Node

### PR DESCRIPTION
**Pull request by zakstam**
Added missing documentation for ReferenceRect node

### CODE REVIEW

File: doc/classes/ReferenceRect.xml [hunk 1]
```diff
@@ -4,14 +4,15 @@
 		Reference frame for GUI.
 	</brief_description>
 	<description>
-		Reference frame for GUI. It's just like an empty control, except a red box is displayed while editing around its size at all times.
+		Reference frame for GUI. It's just like an empty control, except a [member border_color] is displayed while editing around its size at all times.
```
Comment (StraToN) [inline doc/classes/ReferenceRect.xml]: I agree with the change need, although it is unclear what is displayed here: to keep the information that a box is displayed, I'd go with something like `except an outline/box colored with [member border_color] is displayed`.
What is displayed exactly btw? It is an outline (hollow) or a box (full)?

**Review by StraToN — CHANGES_REQUESTED** [review]

File: doc/classes/ReferenceRect.xml (hunk 1 above)
Comment (zaksnet) [reply doc/classes/ReferenceRect.xml]: It's just an outline border. I believe this was intended to emphasize the currently active control

**Review by StraToN — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.1.2.

**Pull request merged**